<launch> To run this launch file, change the<!-- define how many fake tf nodes you want to create -->
    <!-- this number should be the same as the number of <node></node> tags below -->
    <arg name="fake_tf_node_name_1" default="HoloLens" />
    <arg
        name="fake_tf_node_name_2" default="Camera" />
    <arg name="fake_tf_node_name_3"
        default="object_1" />
    <arg name="fake_tf_node_name_4" default="object_2" />


    <!-- <rosparam param="frame_name">sensor_1</rosparam> -->
    <node
        pkg="posehub_tools" type="fake_tf_node" name="fake_node_$(arg fake_tf_node_name_1)"
        output="screen">
        <param name="frame_name" value="$(arg fake_tf_node_name_1)" />
    </node>

    <!-- <rosparam param="frame_name">sensor_2</rosparam> -->
    <node
        pkg="posehub_tools" type="fake_tf_node" name="fake_node_$(arg fake_tf_node_name_2)"
        output="screen">
        <param name="frame_name" value="$(arg fake_tf_node_name_2)" />
    </node>

    <node
        pkg="posehub_tools" type="fake_tf_node" name="fake_node_$(arg fake_tf_node_name_3)"
        output="screen">
        <param name="frame_name" value="$(arg fake_tf_node_name_3)" />
    </node>

    <node
        pkg="posehub_tools" type="fake_tf_node" name="fake_node_$(arg fake_tf_node_name_4)"
        output="screen">
        <param name="frame_name" value="$(arg fake_tf_node_name_4)" />
    </node>

    <node
        pkg="posehub_tools" type="fake_static_ref_node" name="fake_static_ref_node"
        output="screen">
        <param name="frame_name" value="ref_1" />
    </node>

    <!-- <node pkg="posehub_tools" type="fake_sensor_noise_node" name="fake_sensor_noise_node"
        output="screen">
        <param name="sensor_name" value="$(arg fake_tf_node_name_1)" />
    </node> -->

    <node pkg="posehub_tools"
        type="fake_sensor_noise_node" name="fake_sensor_noise_node"
        output="screen">
    </node>

    <node pkg="rviz" type="rviz" name="rviz"
        args="-d $(find posehub_tools)/rviz/config.rviz"
        output="screen" />
</launch>